### DISCUSSION TITLE
I've made a discussion board!

### DISCUSSION THREAD

**Discussion by bobturneruk** [AFM-SPM/home#1] [General]
Hi @alicepyne and @paulmueller. Picking up on email conversations etc. I've set up this repository (you can't have an organisation-level discussion board, it seems). I suggest we maybe keep discussions here software focussed so as not to put off more general chat on the RMS group. I guess we'll see how it goes. I've upgraded this organisation to have unlimited repositories and seats.

**Comment by paulmueller** [comment]
Could you please pin the "home" repository on the organization page? There should be an option somewhere in the settings.

**Comment by bobturneruk** [comment]
Done that (had to google it!).
